package.json

```diff
@@ -36,6 +36,13 @@
   },
   "devDependencies": {
     "eslint": "^7.31.0",
-    "stylelint-test-rule-tape": "^0.2.0"
+    "jest": "^29.5.0",
```

Comment:
But `stylelint-test-rule-tape` is archived https://github.com/stylelint/stylelint-test-rule-tape and uses an old (6.9.0) stylelint version, which in turn uses an old postcss version (5.0.20)

I thought of just copying `stylelint-test-rule-tape` code into the project, but it uses old stylelint api `createRuleTester` https://github.com/stylelint/stylelint-test-rule-tape/blob/master/index.js#L16 that since then has been removed: https://github.com/stylelint/stylelint/issues/4267.

In the removal thread [`jest-preset-stylelint`](https://github.com/stylelint/jest-preset-stylelint) has been recommended as a replacement: https://github.com/stylelint/stylelint/issues/4267#issuecomment-573344678